api/src/routes/schedpredict.routes.ts

```diff
@@ -0,0 +1,32 @@
+import express from 'express';
+import { PredictScheduleController } from '../controllers/schedpredict.controller';
+
+const router = express.Router();
+const predictscheduleController: PredictScheduleController = new PredictScheduleController();
+
+/**
+ * Predict Class Size
+ * include a function to handle the logic for predicting class sizes
+ * @param {*} req
+ * @param {*} res
+ * @return {*}
+ */
+const predict_class_sizes = async (req, res) => {
+  // Validate request
+  if (!req.body.courses || !req.body.csc_to_seng_ratio || !req.body.class_year_split) {
+    res.status(400).send({ message: 'Content can not be empty!' });
+    return;
+  }
+
+  const { courses, csc_to_seng_ratio, class_year_split } = req.body;
+
+  try {
+    const response = await predictscheduleController.predict_class_sizes(courses, csc_to_seng_ratio, class_year_split);
+    res.status(200).send(response);
+  } catch (err) {
+    res.status(401).send({ message: err });
+  }
+};
+router.post('/predict', predict_class_sizes);
```

Comment:
From the api spec as well. currently this is only supplying the route of `/predict` in the spec its highlighted as ` "/schedule/predict_class_sizes": `.
In addition you havnt supplied this router function to the app. It must be added in the app.ts
https://github.com/SENG-499-Company-3/backend/blob/dev/api/src/app.ts
Notice `app.use('/schedule', schedule);` is supplying all schedule router functions to `/schedule` so that the forward ref `/schedule/create` is a valid link